js/controller.js

```diff
@@ -879,10 +894,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     /**
      * The function is called when we want to change audio track
      * @fires OO.EVENTS.SET_CURRENT_AUDIO
-     * @param {String} id - the id of the audio track to activate
+     * @param currentTrack {{id: String, label: String, lang: String}} - current active audio track
      */
-    setCurrentAudio: function(id) {
-      this.mb.publish(OO.EVENTS.SET_CURRENT_AUDIO, id);
+    setCurrentAudio: function(currentTrack) {
+      this.mb.publish(OO.EVENTS.SET_CURRENT_AUDIO, currentTrack);
```

Comment:
Why do we now pass the whole track? Just curious

Reply:
Later on we need to save the whole track to local storage to be able to restore it.